### PR TITLE
Merge branch 'is-a-dev:main' into main

### PR DESCRIPTION
**Pull request by vncsmnl**
To make our job easier, please spend time to review your application before submitting. 

#### Add a x between [] if you meet that requirement example `[x]`
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. **This requirement can be raised if the domain you're registering is for emails**
  - Link to website: 
- [x] The website is reachable.
- [x] The CNAME record doesn't contain any slash.
- [x] There's enough information at the `owner` field.
   - You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.
 
 #### Feel free to join our discord server for more updates or any help related to the service :D - https://discord.gg/PZCGHz4RhQ

### CODE REVIEW

File: domains/vinicius.json [hunk 1]
```diff
@@ -6,6 +6,6 @@
     "email": "59481808+vncsmnl@users.noreply.github.com"
   },
   "record": {
-    "CNAME": "viniciusmanoel.ml"
+    "CNAME": "vncsmnl.github.io/ProfileCard"
```
Comment (WilliamDavidHarrison) [inline domains/vinicius.json:9]: Please remove the `/profilecard`

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/vinicius.json [hunk 2]
```diff
@@ -6,6 +6,6 @@
     "email": "59481808+vncsmnl@users.noreply.github.com"
```
Comment (WilliamDavidHarrison) [inline domains/vinicius.json:6]: Sorry, you can't use this email as we cannot contact you through a noreply email.

File: domains/vinicius.json [hunk 3]
```diff
@@ -6,6 +6,6 @@
...
   },
   "record": {
-    "CNAME": "viniciusmanoel.ml"
+    "CNAME": "vncsmnl.github.io"
```
Comment (WilliamDavidHarrison) [inline domains/vinicius.json:9]: 404?

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/vinicius.json (hunk 3 above)
Comment (vncsmnl) [reply domains/vinicius.json:9]: fixed

File: domains/vinicius.json (hunk 2 above)
Comment (vncsmnl) [reply domains/vinicius.json]: fixed

File: domains/vinicius.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (vncsmnl) [reply domains/vinicius.json:9]: fixed

**Review by wdhdev — APPROVED** [review]

**Comment by vncsmnl** [comment]
fixed

**Review by wdhdev — APPROVED** [review]

**Pull request closed without merging**